packages/react-aria-components/test/NumberField.test.js

```diff
@@ -99,4 +99,29 @@ describe('NumberField', () => {
     expect(group).not.toHaveAttribute('data-focus-visible');
     expect(group).not.toHaveClass('focus');
   });
+
+  it('should support render props', () => {
+    let {getByRole} = render(
+      <NumberField defaultValue={1024} minValue={0}>
+        {({decrementToMin}) => (
```

Comment:
this is also interesting, I'm not sure we meant to expose functions like this, I'll bring it up with the team

Reply:
@auskast  The team discussed this and are leaning towards omitting the state setter methods since ideally users would use the controlled props + external state to manage things like this. We were interested if you had any use cases that benefited from having access to these setters, if so we'd be open to discussing leaving these setters available as is

Reply:
no i don't need those methods specifically, i was just picking some random render props to use to demonstrate that it works. i can use `minValue` and `maxValue` instead if that is better for you.